release/release.go

```diff
@@ -622,6 +679,40 @@ func LatestRC(ctx context.Context, owner, repo, k8sVersion string, client *githu
 
 }
 
+// rke2ChartVersion will return the version of the rke2 chart from the chart versions file
+func rke2ChartsVersion(branchVersion string) (map[string]chart, error) {
+
+	chartsMap := make(map[string]chart)
+	c := charts{}
+	chartVersionsURL := "https://raw.githubusercontent.com/rancher/rke2/" + branchVersion + "/charts/" + rke2ChartsVersionsFile
+	resp, err := http.Get(chartVersionsURL)
```

Comment:
Can you use this function? https://github.com/rancher/ecm-distro-tools/blob/master/http/http.go#L8